libs/build_met.sh

```diff
@@ -0,0 +1,136 @@
+#!/bin/bash                                                                                                                              
+
+set -eux
+
+name="met"
+version=${1:-${STACK_met_version}}
+release_date=${2:-${STACK_met_release_date}}
+
+cd  ${HPC_STACK_ROOT}/${PKGDIR:-"pkg"}
+software=$name-$version.$release_date
+pkg_name=$name-$version
+url="https://github.com/dtcenter/MET/releases/download/v$version/$software.tar.gz"
+[[ -d $software ]] || ( $WGET $url; tar -xf $software.tar.gz )
+[[ -d $pkg_name ]] && cd $pkg_name || ( echo "$pkg_name does not exist, ABORT!"; exit 1 )
+
+
+# Hyphenated version used for install prefix
+compiler=$(echo $HPC_COMPILER | sed 's/\//-/g')
+mpi=$(echo $HPC_MPI | sed 's/\//-/g')
+
+export LMOD_EXACT_MATCH="no"
+
+if $MODULES; then
+    
+    set +x
+    source $MODULESHOME/init/bash
+    module load hpc-$HPC_COMPILER
+    [[ -z $mpi ]] || module load hpc-$HPC_MPI
+    module load hdf5
+    module load netcdf
+    module load bufr
+    module load g2c
+    module load zlib
+    module load jasper
```

Comment:
g2c depends on jasper, so it should be loaded before g2c.

Reply:
Done.